File: Addin/NET48/Source/TinyPlot2D/FlexDlgUserControl/SaveAsForm.vb
Imports System
Imports System.Windows.Forms

Namespace FlexDlgUserCtrl
    Public Partial Class SaveAsForm
        Inherits Form

        Public Property FileName As String
            Get
                Return SaveAsTextBox.Text
            End Get

            Set(value As String)
                SaveAsTextBox.Text = value
            End Set
        End Property


        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub btnCancel_Click(sender As Object, e As EventArgs)
            Close()
        End Sub

        Private Sub btnOK_Click(sender As Object, e As EventArgs)
            Close()
        End Sub
    End Class
End Namespace

File: xlcalcnet/Addin/NET48/Source/TinyPlot2D/FlexDlgUserControl/SaveAsForm.cs
using System;
using System.Windows.Forms;

namespace TinyPlot2DCtrl
{
    public partial class SaveAsForm : Form
    {

        public string FileName
        {
            get
            {
                return SaveAsTextBox.Text;
            }

            set
            {
                SaveAsTextBox.Text = value;
            }
        }


        public SaveAsForm()
        {
            InitializeComponent();
        }

        private void btnCancel_Click(object sender, EventArgs e)
        {
            Close();
        }

        private void btnOK_Click(object sender, EventArgs e)
        {
            Close();
        }
    }
}
